dev/io.openliberty.microprofile.openapi.4.0.internal_fat/fat/src/io/openliberty/microprofile/openapi40/fat/versionwarning/VersionWarningTest.java

```diff
@@ -96,14 +98,26 @@ public void cleanupApps() throws Exception {
     public void testVersionWarning() throws Exception {
         setOpenAPIVersion("3.0");
 
-        WebArchive war = ShrinkWrap.create(WebArchive.class, "testWarning.war")
+        WebArchive war = ShrinkWrap.create(WebArchive.class, APP_NAME + ".war")
                                    .addPackage(VersionWarningApplication.class.getPackage());
 
         server.setMarkToEndOfLog();
         ShrinkHelper.exportDropinAppToServer(server, war, SERVER_ONLY);
 
+        //Assert the version warnings appear on app startup
+        assertVersionWarnings("Failed to find warning on app startup");
+
+        //Assert the version warnings appear on app restart
+        server.setMarkToEndOfLog();
+        Assert.assertTrue("failed to restart the test application", server.restartDropinsApplication(APP_NAME + ".war"));
+        server.findStringsInLogsAndTraceUsingMark("CWWKT0016I:.*" + APP_NAME);
```

Comment:
Don't want to look for this message, `restartDropinsApplication` should cover it.